packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.stories.tsx

```diff
@@ -0,0 +1,327 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import {
+  Alert,
+  AlertTitle,
+  Box,
+  Button,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  InputLabel,
+  OutlinedInput,
+  RadioGroup,
+  Radio,
+  Select,
+  Typography,
+} from "@mui/material";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+
+import FormMdx from "./Form.mdx";
+
+export default {
+  title: `MUI Components/Forms/Form`,
+  component: OutlinedInput,
+  parameters: {
+    docs: {
+      page: FormMdx,
+    },
+  },
+  argTypes: {
+    formAlert: {
+      control: "boolean",
+      defaultValue: null,
+    },
+    formTitle: {
+      control: "text",
+      defaultValue: null,
+    },
+    formTitleSize: {
+      control: "select",
+      options: ["h1", "h2", "h3", "h4"],
+      defaultValue: "h4",
+    },
+    formHint: {
+      control: "text",
+      defaultValue: null,
+    },
+    fieldGroupAlert: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    fieldGroupOne: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    fieldGroupOneTitle: {
+      control: "text",
+      defaultValue: false,
+    },
+    fieldGroupOneHint: {
+      control: "text",
+      defaultValue: false,
+    },
+    fieldGroupTwo: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    fieldGroupTwoTitle: {
+      control: "text",
+      defaultValue: false,
+    },
+    fieldGroupTwoHint: {
+      control: "text",
+      defaultValue: false,
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+const Template: Story = (args) => {
+  const {} = args;
+  return (
+    // <Form>
+    <Box
+      component="form"
+      sx={(theme) => ({
+        maxWidth: theme.mixins.maxWidth,
+      })}
+    >
+      {/* <FormHeader> */}
+      {args.formTitle && (
+        <Box
+          component="div"
+          sx={(theme) => ({
+            marginBottom: theme.spacing(6),
+          })}
+        >
```

Comment:
Is there a better way to write this using `useTheme`? I'd like to promote that method in our stories over functions in `sx`.

Reply:
afaik this is the best way to access theme variables for `<Box>`, since it doesn't have default styles or slots. I don't like the `style`-type application either, though.

I'd like us to get `<Form>` wrapped/exported asap so that downstream teams aren't doing this themselves.